### ISSUE TITLE
train with an outer product as the middle tine

### ISSUE THREAD

**Issue by justin2004** [phantomics/april#217]
```
      ⎕←m←2 3 3 ⍴ ⍳ 18
 1  2  3
 4  5  6
 7  8  9
        
10 11 12
13 14 15
16 17 18
      (⍳∘≢(∘.⌷)⊂) m
┌─────┬────────┐
│1 2 3│10 11 12│
│4 5 6│13 14 15│
│7 8 9│16 17 18│
└─────┴────────┘
```

```
APRIL> (april-f "m←2 3 3 ⍴ ⍳ 18")
#3A(((1 2 3) (4 5 6) (7 8 9)) ((10 11 12) (13 14 15) (16 17 18)))
APRIL> (april-f "(⍳∘≢(∘.⌷)⊂) m")
Value to left of function statement.
   [Condition of type SIMPLE-ERROR]
```

**Comment by justin2004** [comment]
oh, i noticed that this works and is equivalent:
```
APRIL> (april-f "(⍳∘≢∘.⌷⊂) m")
 1 2 3  10 11 12
 4 5 6  13 14 15
 7 8 9  16 17 18
#(#2A((1 2 3) (4 5 6) (7 8 9)) #2A((10 11 12) (13 14 15) (16 17 18)))
```

then i think this issue is a duplicated of #209 ?

**Comment by phantomics** [comment]
This is fixed along with #209.
